provider/image.go

```diff
@@ -942,3 +943,16 @@ func mapDockerignore(dockerfile string) string {
 	// Return the default dockerignore name.
 	return ignore
 }
+
+func SSHOptsToSlice(sshOptsFromConf string) []string {
+
+	trimmedOpts := strings.Trim(sshOptsFromConf, "[]")
+	slicedOpts := strings.Split(trimmedOpts, ",")
+	sshopts := []string{}
+
+	for _, val := range slicedOpts {
+		val = strings.ReplaceAll(val, "\"", "")
+		sshopts = append(sshopts, val)
+	}
+	return sshopts
+}
```

Comment:
I believe that we just encode this into JSON. I think we should just decode it as json as well (`json.Unmarshal`). I'm not sure how this works with computed values in provider inputs.